### PR TITLE
Fix creation of .env file composer create project

### LINKED ISSUE #118: .env file not found at composer create project
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 2.0.0-alpha4

#### Actual Behavior

When you run composer create project, you get an error that the .env file was not found.

#### Expected Behavior

The normal behavior should be that the `.env.dist` should be copied to `.env`.

#### Steps to Reproduce

Run `composer create-project sulu/sulu-minimal:2.0.0-alpha4`

### PR DESCRIPTION
**Pull request by alexander-schranz**
Related PR: https://github.com/sulu/sulu/pull/4206

Also moved JS tests to CircleCi and added composer create project as test

fixes #118

### CODE REVIEW

**Comment by danrot** [comment]
This is the fix for #118. And I would not move the NPM tests to circle ci, since seem to have install the composer dependencies there as well...

**Comment by alexander-schranz** [comment]
@danrot travis is splitted in 2 test cases one using env variables and one uses the .env file with composer create-project. not sure if we should remove the first one?

File: .travis.yml [hunk 1]
```diff
@@ -6,48 +6,67 @@ cache:
...
+  # Install project over composer create and set env variables in .env file
+  - |
+    if [[ $COMPOSER_CREATE = true ]]; then
+        composer create-project sulu/sulu-minimal ../test-create-project-dir --repository="{\"type\":\"path\",\"url\":\"./\"}" --stability=dev -n
+        cd ../test-create-project-dir
+        php -r "file_put_contents('.env', str_replace('DATABASE_URL=#mysql://db_user:db_password@127.0.0.1:3306/db_name', 'DATABASE_URL=mysql://root:@127.0.0.1:3306/sulu_test', file_get_contents('.env')));"
+        php -r "file_put_contents('.env', str_replace('APP_ENV=dev', 'APP_ENV=test', file_get_contents('.env')));"
+    fi
+  - ls -lh
+  - pwd
```
Comment (danrot) [inline .travis.yml]: Are this two commands remaining from testing?

**Review by danrot — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -6,48 +6,67 @@ cache:
...
+    # Without ENVIRONMENT variables
+    - php: 7.2
+      env:
+        - COMPOSER_CREATE: true
+        - NPM_BUILD: true
 
 script:
+  # Test composer create project
+  - echo $TRAVIS_PULL_REQUEST_BRANCH
+  - echo $TRAVIS_PULL_REQUEST_SLUG
```
Comment (danrot) [inline .travis.yml]: These are probably also remainders from testing, right?

**Review by danrot — CHANGES_REQUESTED** [review]

**Review by danrot — APPROVED** [review]

**Pull request merged**